Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/Dates.kt

```diff
@@ -0,0 +1,7 @@
+package de.rki.coronawarnapp.util
+
+object Dates {
+    private const val MILLIS_PER_DAY = 24 * 3600 * 1000
+
+    fun numberOfDays(t0: Long, t1: Long) = ((t1 - t0) / MILLIS_PER_DAY).toInt()
```

Comment:
Please replace custom time logic with framework classes whereever possible, i.e. here something like `Duration(Instant.now(),Instant.now()).standardDays`

Reply:
Likely remove the `Dates` class and use framework classes where `Dates.numberOfDays` is currently used.